service/pokemon_service.go

```diff
@@ -0,0 +1,25 @@
+package service
+
+import (
+	"WizelineApi/domain/model"
+	"errors"
+)
+
+//Servicio utilizado para buscar por ID dentro de un slice del modelod e Pokemon
+func GetPokemonById(pokemons []model.Pokemon, id int) (model.Pokemon, error){
+	t_pkm := model.Pokemon{}
+
+	for _,pk := range pokemons{
+		if pk.ID == id {
+			t_pkm = pk
+			break
+		}
+	}
+
+	if t_pkm.Name == "" {
+		return t_pkm, errors.New("Couldnt find pokemon")
```

Comment:
Instead of dynamically creating and using errors, better define them beforehand and just return them here.

It's up to you to decide whether you define these errors in this package or define a new errors package (or something similar).